### PR TITLE
Responsive web

### PR DESCRIPTION
**Pull request by GerardoSabetta**
This branch contains my custom grid

### CODE REVIEW

File: topic0/assets/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+.g-row:before,.g-row:after {
+    content:"";
+    display:table;
+    clear:both;
+}
+
+[class*='g-col-'] {
+    float:left;
+    min-height: 1px;
+    width: 12.5%;
```
Comment (troianoandres) [inline topic0/assets/styles/style.css]: Since all cols has width, is this really needed?

File: topic0/assets/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+.img-description {
+    position:absolute;
+    background: white;
+    height: 100%;
+    top:0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    visibility: hidden;
```
Comment (troianoandres) [inline topic0/assets/styles/style.css]: Why do we actually needed visibility if opacity is set?

**Comment by troianoandres** [comment]
Once you want merge this PR. You are the one in charge of merging PRs not me

**Comment by GerardoSabetta** [comment]
I know, but I'd like to reorganize my topic 0 file structure before merging is there a problem if I don't merge in time?

**Pull request merged**